SafeExamBrowser.Browser/Handlers/KeyboardHandler.cs

```diff
@@ -20,6 +20,9 @@ internal class KeyboardHandler : IKeyboardHandler
 		internal event ActionRequestedEventHandler ZoomInRequested;
 		internal event ActionRequestedEventHandler ZoomOutRequested;
 		internal event ActionRequestedEventHandler ZoomResetRequested;
+		internal event System.EventHandler<bool> TabPressed;
```

Comment:
Fixed that by creating `TabPressedEventHandler`.

I'm no fan of `object sender` either, but I'm also no fan of dozens of event handler declaration files. Would prefer a generic way to declare event signatures, like

```
public delegate void SebEventHandler();
public delegate void SebEventHandler<T1>(T1 item1);
public delegate void SebEventHandler<T1, T2>(T1 item1, T2 item2);
public delegate void SebEventHandler<T1, T2, T3>(T1 item1, T2 item2, T3 item3);
...
```

This would require us to document clearly the meaning of the `itemX` parameters in the event declarations.

Of course I can live with SEB's way of declaring event handlers :-)

Reply:
Yes, unfortunately there's no nice and easy way which does it all. Great, that's perfect!